guide/english/certifications/javascript-algorithms-and-data-structures/basic-algorithm-scripting/mutations/index.md

```diff
@@ -89,6 +89,41 @@ Grab the second string, lowercase and turn it into an array; then make sure _eve
 *   <a href='http://forum.freecodecamp.com/t/javascript-string-prototype-split/15944' target='_blank' rel='nofollow'>Array.split()</a>
 *   <a href='http://forum.freecodecamp.com/t/javascript-array-prototype-every/14287' target='_blank' rel='nofollow'>Array.every()</a>
 
+## ![:rotating_light:](https://forum.freecodecamp.com/images/emoji/emoji_one/rotating_light.png?v=3) Advanced Code Solution:
+
+**Procedural**
+
+```js
+
+    function mutation(arr) {
+      let re = new RegExp("(?=.*" + arr[1].toLowerCase().split("").join(')(?=.*') + ")","ig");
```

Comment:
Remove extra leading space and since there is no need for the global flag, remove it from the regular expression
```suggestion
  let re = new RegExp("(?=.*" + arr[1].toLowerCase().split("").join(')(?=.*') + ")","i");
```